cpu/esp8266/startup.c

```diff
@@ -50,6 +51,17 @@ extern uint32_t hwrand (void);
 
 void esp_riot_init(void)
 {
+#ifdef RIOT_VERSION
```

Comment:
Is this so it doesn't get called by `riotboot`?

Reply:
`riotboot` is'nt used. It's just a leftover when I moved that code from vendor code to here. Embedding the code in `#ifdef RIOT_VERSION ... #endif` isn't necessary any longer. I removed it.